### PR TITLE
wdio-reporter: Type definitions

### PR DESCRIPTION
**Pull request by lamkovod**
## Proposed changes

Typings for WDIOReporter.
+
added `pageLoad` to `WebdriverIO.Options`

## Types of changes
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Code example:
```
import WDIOReporter from '@wdio/reporter';

export default class CustomReporter extends WDIOReporter {
   constructor (options: WDIOReporter.Options) {
       options = Object.assign(options, { stdout: true });
       super(options);
   }
};
```
#3572 
### Reviewers: @webdriverio/technical-committee

### CODE REVIEW

File: packages/wdio-reporter/reporter.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    interface Options {
+        configFile: string;
+        logFile: string;
+        logLevel: string;
+        stdout?: boolean;
+    }
+
+    interface Suite {
```
Comment (mgrybyk) [inline packages/wdio-reporter/reporter.d.ts]: Can we somehow reuse Suite, Test, Hook (or at least something) from Webdriverio types

**Comment by mgrybyk** [comment]
May you please also add some very basic types test

File: packages/wdio-reporter/reporter.d.ts (hunk 1 above)
Comment (lamkovod) [reply packages/wdio-reporter/reporter.d.ts]: I don't think that we can do that. 
It's another package

File: scripts/templates/webdriverio.tpl.d.ts [hunk 2]
```diff
@@ -63,6 +63,7 @@ declare namespace WebdriverIO {
         specFileRetries?: number,
         waitforTimeout?: number,
         waitforInterval?: number,
+        pageLoad?: number,
```
Comment (christian-bromann) [inline scripts/templates/webdriverio.tpl.d.ts]:
```suggestion
```

This is not a WebdriverIO option

**Pull request merged**